<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>ExpandThumbnailForm</class>
 <widget class="QWidget" name="ExpandThumbnailForm">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>877</width>
    <height>713</height>
   </rect>
  </property>
  <property name="sizePolicy">
   <sizepolicy hsizetype="Expanding" vsizetype="Expanding">
    <horstretch>0</horstretch>
    <verstretch>0</verstretch>
   </sizepolicy>
  </property>
  <property name="windowTitle">
   <string>ExpandThumbnailForm</string>
  </property>
  <layout class="QGridLayout" name="gridLayout_3">
   <property name="sizeConstraint">
    <enum>QLayout::SetDefaultConstraint</enum>
   </property>
   <item row="0" column="0">
    <layout class="QHBoxLayout" name="horizontalLayout">
     <property name="spacing">
      <number>5</number>
     </property>
     <item>
      <widget class="ImageWidgetInThumbnail" name="lbMaster" native="true"/>
     </item>
     <item>
      <widget class="ImageWidgetInThumbnail" name="lbNG" native="true"/>
     </item>
    </layout>
   </item>
   <item row="0" column="1">
    <layout class="QVBoxLayout" name="verticalLayout">
     <item>
      <widget class="QGroupBox" name="gbIngomation">
       <property name="sizePolicy">
        <sizepolicy hsizetype="Fixed" vsizetype="Fixed">
         <horstretch>0</horstretch>
         <verstretch>0</verstretch>
        </sizepolicy>
       </property>
       <property name="layoutDirection">
        <enum>Qt::LeftToRight</enum>
       </property>
       <property name="title">
        <string>Infomation</string>
       </property>
       <property name="flat">
        <bool>false</bool>
       </property>
       <property name="checkable">
        <bool>false</bool>
       </property>
       <layout class="QGridLayout" name="gridLayout_2">
        <item row="0" column="1" rowspan="2">
         <layout class="QGridLayout" name="gridLayout">
          <item row="0" column="0">
           <widget class="QLabel" name="label">
            <property name="text">
             <string>Target (X,Y)</string>
            </property>
            <property name="alignment">
             <set>Qt::AlignCenter</set>
            </property>
           </widget>
          </item>
          <item row="4" column="0">
           <widget class="QLabel" name="label_2">
            <property name="text">
             <string>Master (X,Y)</string>
            </property>
            <property name="alignment">
             <set>Qt::AlignCenter</set>
            </property>
           </widget>
          </item>
          <item row="0" column="1">
           <widget class="QLabel" name="lbTargetPos">
            <property name="text">
             <string>Target (X,Y)</string>
            </property>
            <property name="alignment">
             <set>Qt::AlignCenter</set>
            </property>
           </widget>
          </item>
          <item row="6" column="0">
           <widget class="QLabel" name="label_3">
            <property name="text">
             <string>NGCount</string>
            </property>
            <property name="alignment">
             <set>Qt::AlignCenter</set>
            </property>
           </widget>
          </item>
          <item row="6" column="1">
           <widget class="QLabel" name="lbNGCount">
            <property name="text">
             <string>NGCount</string>
            </property>
            <property name="alignment">
             <set>Qt::AlignCenter</set>
            </property>
           </widget>
          </item>
          <item row="9" column="0">
           <widget class="QLabel" name="label_4">
            <property name="text">
             <string>FKeyType</string>
            </property>
            <property name="alignment">
             <set>Qt::AlignCenter</set>
            </property>
           </widget>
          </item>
          <item row="4" column="1">
           <widget class="QLabel" name="lbMasterPos">
            <property name="text">
             <string>Master (X,Y)</string>
            </property>
            <property name="alignment">
             <set>Qt::AlignCenter</set>
            </property>
           </widget>
          </item>
          <item row="7" column="0" colspan="2">
           <widget class="Line" name="line_3">
            <property name="orientation">
             <enum>Qt::Horizontal</enum>
            </property>
           </widget>
          </item>
          <item row="5" column="0" colspan="2">
           <widget class="Line" name="line_2">
            <property name="orientation">
             <enum>Qt::Horizontal</enum>
            </property>
           </widget>
          </item>
          <item row="3" column="0" colspan="2">
           <widget class="Line" name="line">
            <property name="orientation">
             <enum>Qt::Horizontal</enum>
            </property>
           </widget>
          </item>
          <item row="9" column="1">
           <widget class="QLabel" name="lbFKeyType">
            <property name="text">
             <string>FKeyType</string>
            </property>
            <property name="alignment">
             <set>Qt::AlignCenter</set>
            </property>
           </widget>
          </item>
          <item row="10" column="0">
           <widget class="QLabel" name="label_5">
            <property name="text">
             <string>Key code</string>
            </property>
            <property name="alignment">
             <set>Qt::AlignCenter</set>
            </property>
           </widget>
          </item>
          <item row="10" column="1">
           <widget class="QComboBox" name="comboBoxKeyCode">
            <property name="editable">
             <bool>true</bool>
            </property>
            <item>
             <property name="text">
              <string>No key</string>
             </property>
            </item>
            <item>
             <property name="text">
              <string>F1</string>
             </property>
            </item>
            <item>
             <property name="text">
              <string>F2</string>
             </property>
            </item>
            <item>
             <property name="text">
              <string>F3</string>
             </property>
            </item>
            <item>
             <property name="text">
              <string>F4</string>
             </property>
            </item>
            <item>
             <property name="text">
              <string>F5</string>
             </property>
            </item>
            <item>
             <property name="text">
              <string>F6</string>
             </property>
            </item>
            <item>
             <property name="text">
              <string>F7</string>
             </property>
            </item>
            <item>
             <property name="text">
              <string>F8</string>
             </property>
            </item>
            <item>
             <property name="text">
              <string>F9</string>
             </property>
            </item>
            <item>
             <property name="text">
              <string>F10</string>
             </property>
            </item>
           </widget>
          </item>
          <item row="11" column="1">
           <widget class="QPushButton" name="pushButtonSetKey">
            <property name="text">
             <string>SetKey/Code</string>
            </property>
           </widget>
          </item>
         </layout>
        </item>
       </layout>
      </widget>
     </item>
     <item>
      <spacer name="verticalSpacer">
       <property name="orientation">
        <enum>Qt::Vertical</enum>
       </property>
       <property name="sizeHint" stdset="0">
        <size>
         <width>20</width>
         <height>40</height>
        </size>
       </property>
      </spacer>
     </item>
    </layout>
   </item>
  </layout>
 </widget>
 <customwidgets>
  <customwidget>
   <class>ImageWidgetInThumbnail</class>
   <extends>QWidget</extends>
   <header>ImageWidgetInThumbnail.h</header>
   <container>1</container>
  </customwidget>
 </customwidgets>
 <resources/>
 <connections/>
</ui>
